bigbone-rx/src/main/kotlin/social/bigbone/rx/RxAppMethods.kt

```diff
@@ -12,7 +12,6 @@ import social.bigbone.api.method.AppMethods
  * @see <a href="https://docs.joinmastodon.org/methods/apps/">Mastodon apps API methods</a>
  */
 class RxAppMethods(client: MastodonClient) {
-
     private val appMethods = AppMethods(client)
```

Comment:
I also really don’t like the missing space. 😄

Reply:
Rule name here would be `standard:no-empty-first-line-in-class-body`.

Reply:
How do you feel about that one? @andregasser @bocops 

I could live with the new version, but I know I’d write it differently if I were to write an application on my own.

Reply:
Thanks for the feedback. I will disable the rule.